.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: Test
+on: [push, pull_request]
+
+jobs:
+  test:
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        go-version: ["1.16.x", "1.17.x"]
+        platform: [ubuntu-20.04]
+    runs-on: ${{ matrix.platform }}
+    env:
+      GO111MODULE: auto
+    steps:
+    - name: Install Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+    - name: Set env
+      shell: bash
+      run: |
+        echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+        echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        path: src/github.com/LK4D4/vndr
+    - name: Gofmt
+      run: test -z "$(gofmt -s -l . | tee /dev/stderr)"
+      working-directory: src/github.com/LK4D4/vndr
+    - name: Test
+      run: pwd && go test -v .
+      working-directory: src/github.com/LK4D4/vndr
+    - name: Integration tests
+      run: |
+        go install .
+        go test -v test/vndr_test.go
+      working-directory: src/github.com/LK4D4/vndr
+  lint:
+    name: Linters
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        go-version: ["1.16.x"]
```

Comment:
Kept this at go1.16 for now; on go1.17 it gets confused by the `// +build` examples in the docs (will need to have a look if we can add support for go1.17-style `//go:build` tags 😞